apps/src/lib/client/rpc.rs

```diff
@@ -1293,20 +1295,37 @@ pub async fn query_storage_value<T>(
 where
     T: BorshDeserialize,
 {
+    let (value, _proof) =
+        query_storage_value_bytes(client, key, None, false).await;
+    match value {
+        Some(v) => match T::try_from_slice(&v[..]) {
+            Ok(value) => return Some(value),
+            Err(err) => eprintln!("Error decoding the value: {}", err),
+        },
+        None => return None,
+    }
+    cli::safe_exit(1)
```

Comment:
```suggestion
    if let Some(Err(err)) =  value {
        eprintln!("Error deconding the value: {}", err);
        cli::safe_exit(1)
    } 
    value.map(|inner| inner.unwrap())
```

Reply:
I renamed these variables. `value` is used twice.